spec.bs

```diff
@@ -418,19 +418,29 @@ try {
     not) be used by a [=Relying Party=] for other authentication flows (e.g.,
     login).
 
-: <dfn>Steps to determine if a credential is SPC-enabled</dfn>
-:: An as-yet undefined process by which a user agent can, given a [=credential
-    ID=], determine if the credential represented by that ID is an [=SPC
-    Credential=].
+    Note: The current version of this specification requires the [=Relying
+          Party=] to explicitly opt in for a credential to be used in SPC in
```

Comment:
```suggestion
          Party=] to explicitly opt in for a credential to be used 
```

Reply:
Since start of sentence is "this specification" I think "in SPC in" can be dropped.

Reply:
Done.